src/sfctl/custom_chaos_schedule.py

```diff
@@ -0,0 +1,163 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Custom commands for the Service Fabric chaos schedule test service"""
+
+def parse_time_of_day(time_of_day):
+    """Parse a TimeOfDay from string"""
+    from azure.servicefabric.models.time_of_day import (
+        TimeOfDay
+    )
+
+    if not time_of_day:
+        return None
+
+    hour = time_of_day.get("Hour")
+    minute = time_of_day.get("Minute")
+
+    if hour is None or minute is None:
+        return None
+
+    return TimeOfDay(hour, minute)
+
+def parse_time_range(time_range):
+    """Parse a TimeRange from string"""
+    from azure.servicefabric.models.time_range import (
+        TimeRange
+    )
+
+    if time_range is None:
+        return None
+
+    start_time = parse_time_of_day(time_range.get("StartTime"))
```

Comment:
What should happen if they get None from parse_time_of_day? Can you add a comment here for that? Thanks!